packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -33,6 +33,11 @@ contract Reserve is
     uint128 timestamp;
   }
 
+  struct ExchangeSpender {
```

Comment:
Why the top-level struct? Why not have the fields be top-level structs themselves?

Reply:
not big difference for me, as they are related wanted to have it in the same structure, we have a tendency on the codebase to leave separate?

Reply:
the automatic generated getters won't work with struct types
my opinion would be to only define a struct if it will be stored multiple times

Reply:
cool, will remove